README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+# Graasp App: Quiz
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
+<img src="https://img.shields.io/github/package-json/v/graasp/graasp-app-quiz?color=17b558" alt="Version"/>
+<a href="https://choosealicense.com/licenses/agpl-3.0/#"><img src="https://img.shields.io/github/license/graasp/graasp-app-quiz?color=00bfff" alt="License"/></a>
+</div>
 
-## Available Scripts
+An app to create interactive quizzes on the Graasp platform.
 
-In the project directory, you can run:
+![App Screenshots](docs/slider.png)
 
-### `npm start`
+The app support multiple types of questions:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- simple text responses
+- multiple answer questions
+- quantitative answers using a slider
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Usage
 
-### `npm test`
+If you wish to run the app in your local environnement you will need to have a `.env.local` file with the following content:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```sh
+# Domain for the react app
+REACT_APP_GRAASP_DOMAIN=localhost
+# ID of the app (can be generated online https://www.uuidgenerator.net/version4)
+REACT_APP_GRAASP_APP_ID=5b321475-ecb8-4df2-9e32-f4a4cd444fa4
+# Enable mock API (to use the app without the graasp backend)
+REACT_APP_ENABLE_MOCK_API=true
+# Address of the API host (graasp backend)
+REACT_APP_API_HOST=localhost:3000
+# Sentry DSN url
+REACT_APP_SENTRY_DSN=""
```

Comment:
Are `""` necessary? Maybe let's set "<sentry-dsn>". I don't remember if we have this check, but it would be better to `if(SENTRY_DSN)` so it still works well with an undefined value.